src/patternfly/components/Table/examples/Table.md

```diff
@@ -125,7 +125,9 @@ import './Table.css'
 | Class | Applied to | Outcome |
 | -- | -- | -- |
 | `.pf-v5-c-table` | `<table>` | Initiates a table element. **Required** |
-| `.pf-v5-c-table__caption` | `<caption>` | Initiates a table caption. |
+| `.pf-v5-c-table__tr` | `<th>` | Initiates a table row. **Required** |
```

Comment:
Doesn't look like we're always adding this class - should it be marked as required?

```suggestion
| `.pf-v5-c-table__tr` | `<tr>` | Initiates a table row. **Required** |
```